new/LIT6389MalkeaTewodros.xml

```diff
@@ -0,0 +1,75 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6389MalkeaTewodros" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">መልክአ፡ ቴዎድሮስ፡</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">Mälkǝʿa Tewodros</title>
```

Comment:
Malkǝʿa

Reply:
Thanks for these! I was transliterating Amharic a la Appleyard, and it muddled me!